tests/hedgehog_test_runner.py

```diff
@@ -4,8 +4,12 @@
 
 import paramiko
 import yaml
+import json
+
+from pathlib import Path
 
 FULL_REPORT_DIR_PATH = ""
+MAPPING_TEST_TO_GROUP_FILE = "./mapping_test_to_group.yaml"
```

Comment:
how about moving this file into `tests/hedgehog` directory ? 
just to keep all hedgehog related files in the same place

Reply:
Thanks, updated